Makefile

```diff
@@ -7,7 +7,7 @@ PKG = ./pkg/...
 BIN ?= $(OUTPUT_DIR)/$(APP)
 KUBECTL_BIN ?= $(OUTPUT_DIR)/kubectl-$(APP)
 
-GO_FLAGS ?= -v -mod=vendor
+GO_FLAGS ?= -mod=vendor
```

Comment:
I would recommend to leave the `-v` flag, please. It helps to read CI output and differentiate what's what.